src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Ball {
+    private static final int MAX_NUMBER = 100;
```

Comment:
move this to class Lottery

Reply:
are you suggesting to delete get method for constant?

Reply:
and move this constant to lottery and to not add get method for it?